flixel/text/FlxBitmapText.hx

```diff
@@ -52,6 +52,7 @@ class FlxBitmapText extends FlxSprite
 	/**
 	 * Specifies how the text field should align text.
 	 * JUSTIFY alignment isn't supported.
+	 * Note: For alignment to work, 'autoSize' must be set to false.
```

Comment:
Hm.. Technically, alignment probably works, it's just that it doesn't look any different with the width being as small as possible?

I guess a regular FlxText behaves the same way?